job/models.py

```diff
@@ -569,6 +569,7 @@ class SiteOutputs(BaseModel, models.Model):
     )
 
 """
+# TODO: Remove this commented out text? 
```

Comment:
@hdunham and @Bill-Becker Do either of you see a reason to keep this commented out `SiteOutputs` content?

Reply:
removed in another branch's merge into develop, I believe